src/Validation/Factory.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/* Copyright (c) 2017 Richard Klees <richard.klees@concepts-and-training.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\Validation;
+
+/**
+ * Factory for creating restrictions.
+ */
+class Factory {
+	// COMBINATORS
+
+	/**
+	 * Get a restriction that sequentially checks the supplied restrictions.
+	 *
+	 * The new restriction tells the problem of the first violated restriction.
+	 *
+	 * @param   Restriction[]   $others
+	 * @return  Restriction
+	 */
+	public function seq(array $others);
```

Comment:
I have the same opinion as Timon, I think it is better to be explicit when naming methods than using abbreviations.